frontend/src/test/java/com/google/tripmeout/serialization/testdata/TestDataAccessUtil.java

```diff
@@ -0,0 +1,46 @@
+package com.google.tripmeout.serialization.testdata;
+
+import com.google.common.io.CharStreams;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+
+public final class TestDataAccessUtil {
```

Comment:
If you do end up defining JSON test data inline, this class can likely go away.

Reply:
On second thought, don't worry about making this change here. Without Java support for raw string literals, it becomes difficult to read the escaped values. It's a trade-off between the test JSON itself being readable and being close to its specialized test case.